### DISCUSSION TITLE
'Painting' mode for the colour and delete tools?

### DISCUSSION THREAD

**Discussion by welsmore** [leozide/leocad#813] [Ideas]
I wonder if it's possible to have the paint tool work like photoshop etc, when editing multiple bricks? I guess this would be a preference you could toggle on and off. 

i.e. Let the user hold down the mouse button, then drag over multiple bricks - changing each elements' colour as you mouse over it. Instead of the current repeated clicking that is required.

This would really speed up the colouring of sculpted models, like the turtle here: https://www.warrenelsmore.com/latest/barrierreef . If it could work for the delete tool as well that would allow for digital 'sculpting' of landscapes and other forms. Auto-generating a block of 1x1 bricks to 'chip away at' is fairly easy.

**Comment by nathaneltitane** [comment]
@leozide uncanny timing on this as I was actually preparing to open a similar ticket.
 my l2cu utility does recursive model part, color  nad part+color combination modification throughout the model file (be it ldr or mpd)

would be nice to maybe include an option to recurse when filters are applied?

**Comment by leozide** [comment]
I've added drag paint
